README.md

```diff
@@ -642,31 +577,30 @@ Note that backslashes in double-quoted strings *must* be double-escaped and
 backslashes in single-quoted strings *may* be double-escaped.
 
 ##### `name`
-(**Namevar**: If ommitted, this attribute's value will default to the resource's
-title.)
 
-The package name. This is the name that the packaging system uses internally.
+Specifies the package name. This is the name that the packaging system uses internally. Valid options: String. Default: The resource's title.
 
 ##### `provider`
-The specific backend to use for the `package` resource. Chocolatey is not the
-default provider for Windows so it must be specified (or by using a resource
-default, shown in Usage). Valid options for this provider are `'chocolatey'`.
+
+**Required.** Sets the specific backend to use for the `package` resource. Chocolatey is not the
+default provider for Windows, so it must be specified (or by using a resource
+default, shown in the Usage section above). Valid options: `'chocolatey'`.
 
 ##### `source`
-Where to find the package file. Chocolatey maintains default sources in its
-configuration file that it will use by default. Use this to override the default
-source(s).
 
-Chocolatey accepts different values for source, including accept paths to local
-files/folders stored on the target system, URLs (to OData feeds), and network
-drive paths. Puppet will not automatically retrieve source files for you, and
-usually just passes the value of source to the package installation command.
+Specifies where to find the package file. Use this to override the default
+source(s). Valid options: String of either an absolute path to a local
+directory stored on the target system, a URL (to OData feeds), or a network
+drive path. Chocolatey maintains default sources in its configuration file that it will use by default.
```

Comment:
Should we add a special note here to let folks know this is NOT the path to a file, rather a directory that contains packages when using a CIFS/Directory path as the source?